java/bogdana_maksiuta/src/com/zoolatech/lecture3/tasks/_2/UserAccountValidator.java

```diff
@@ -0,0 +1,33 @@
+package com.zoolatech.lecture3.tasks._2;
+
+import java.util.ArrayList;
+
+public class UserAccountValidator implements Validator {
+    UserAccount userAccount;
+
+    public UserAccountValidator(UserAccount userAccount) {
+        this.userAccount = userAccount;
+    }
+
+    @Override
+    public boolean isValid(UserAccount userAccount) {
+        ArrayList<Validator> userAccountValidators = new ArrayList<>();
```

Comment:
validators can be added to the list in the constructor (note, as per task description you need to pass user account field names, and not values).